src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+import { TodosFilter } from '../TodosFilter/TodoFilter';
+
+export const Footer = () => {
+  const { todos, updateTodos } = useContext(TodosContext);
+
+  const handleClearCompleted = () => {
+    updateTodos(todos.filter(todo => todo.completed !== 'completed'));
+  };
+
+  const amountOfUncompletedTodos = todos.filter(
```

Comment:
Use more clear naming for variables. It's hard to read this name for now. activeTodos or activeTodosCount maybe?